.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: ruby
+rvm:
+  - 2.3.0
+env:
+  - DEFAULT_MAILER_HOST=localhost:1080 SECRET_KEY_BASE=asecretkeybase CAMPAIGN_ENDED=false MAILER_SENDER=mailer@example.com
```

Comment:
ya, i was confused by that as well. the section under what you're pointing me to shows how to run multiple builds by using multiple lines. so even though i was following the first example in which there's one env var per line, it was triggering multiple builds with only that one env var being set so the build would fail.